modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Executor.scala

```diff
@@ -14,13 +14,14 @@ import Scalaz._
 import scalaz.concurrent.Task
 
 object Step {
+  val dhsClient = DhsClientSim(LocalDate.now())
```

Comment:
This needs to be constructed somewhere and being Step an object there are no good places. I guess it could be parameter of the `step` method and be passed up the stack